mammalogy-notes.csl

```diff
@@ -0,0 +1,255 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
```

Comment:
@dlizcano, thanks. The `default-locale="en-US"` here fixes the style to US English. Is that correct? Or is the journal bilingual (English/Spanish)? If the latter, we would typically remove `default-locale` so that users can specify the desired locale to use with the style.

Reply:
I looked through a couple of issues -- while the articles are mostly Spanish, the citations are mostly English, so the default-locale makes sense as is (some Spanish-language references do use Spanish terms, but we can't handle that distinction in CSL)

Reply:
The the journal is bilingual (English/Spanish). 
Thanks for accepting the citation-style.